html/index.html

```
@@ -7,12 +7,17 @@
     <title>YtDownloader</title>
+    
+    <script src="../assets/slimselect.js"></script>
+    <link href="../assets/slimselect.css" rel="stylesheet">
+
     <link rel="stylesheet" href="../assets/css/index.css">
 
     <script src="../translations/i18n-init.js" defer></script>
     <script src="../src/renderer.js" defer></script>
     <script src="../src/index.js" defer></script>
     <script src="../src/common.js" defer></script>
+    </link>
```

HTMLHint (1.9.2)

[error] Tag must be paired, no start tag: [ </link> ]

(tag-pair)